frontend/src/components/mtfi/CurrentFeeVerification.vue

```diff
@@ -1264,15 +840,8 @@ export default {
     hasModelChanged() {
       return !isEqual(this.CCFRIFacilityModel, this.loadedModel);
     },
-    arePrevFeesCorrect() {
-      return !(
-        !this.CCFRIFacilityModel.existingFeesCorrect || this.CCFRIFacilityModel.existingFeesCorrect == '100000001'
-      );
-    },
     isInputVisible(index) {
```

Comment:
Could this one be simplified to: "return !isEmpty(this.CCFRIFacilityModel.childCareTypes[index].feeFrequency);"?

Reply:
The facility model always comes loaded with a selection so the drop down works.. So your code change would always return true